README.md

```diff
@@ -16,74 +16,12 @@
 </p>
 
 <p align="center">
-Dedicated to making your work flow more seamless. Search everything from applications, files, bookmarks, YouTube, Twitter and more. Flow will continue to evolve, designed to be open and built with the community at heart.
```

Comment:
@coderabbitai summarise this release PR based on each merged commit's PR, link each summarised change to the PRs.

Reply:
@coderabbitai summarise this with merged PR's title and link but only PRs that have a milestone of 1.19.0

Reply:
give me the full list

Reply:
Give me the full list summary of the merged PRs for milestone 1.19.0 with labels bug or enhancement, and categorise them based on those two labels.

Reply:
Give me the categorized summary (with enhancements before but fixes) of the merged PRs for milestone 1.19.0 with labels "enhancement" or "bug", and an additional paragraph of all the contributors in it.

Reply:
Give me the categorized summary of the merged PRs for milestone 1.19.0, with enhancements listed before bug fixes